2020/_data/speakers.json

```diff
@@ -126,8 +142,8 @@
       "enabled": "1",
       "name": "Ben Evans",
       "company": "New Relic",
-      "description": "Principal Engineer & JVM Technologies Architect at New Relic, Inc.",
-      "biography": "Ben Evans is Principal Engineer and Architect for JVM technologies at New Relic. Prior to joining New Relic, Ben co-founded jClarity (acquired by Microsoft) and was Chief Architect (Listed Derivatives) at Deutsche Bank. Ben is the author of 5 books - \"The Well-Grounded Java Developer\", \"Java: The Legend\", \"Optimizing Java\" and the recent editions of \"Java in a Nutshell\". He is the track lead for Java / JVM at infoq.com, writes regularly for industry publications and is a frequent speaker at technical conferences worldwide. Ben has been active in Free and Open Source Software for over 20 years, co-founded the AdoptOpenJDK initiative (with Martijn Verburg), and served on the JCP Executive Committee for 6 years.",
+      "description": "Technologist, Writer, & Educator. Java Champion, Java / JVM track lead at InfoQ, bestselling tech author, speaker. ",
```

Comment:
It is not important, but be careful when adding big descriptions, in the "speaker's info box" doesn't appear to ber really nice (`Home > Speakers section`)
![image](https://user-images.githubusercontent.com/2370634/74656953-88789b00-518f-11ea-8ab1-317982e55bdf.png)